### ISSUE TITLE
"Assign Roles" icon in the Essential theme looks like a "lock"

### ISSUE THREAD

**Issue by pccde** [gjb2048/moodle-theme_essential#499]
The icon for "Assign Roles" appears to be a "lock" icon.  I would like to modify or replace it with the previous icon that has a person icon with a plus sign.  Can this be done?

**Comment by gjb2048** [comment]
Almost certainly.  However all enhancements now need to be funded.

**Comment by pccde** [comment]
We might be able to come up with funding.  What amount are we talking about?


Mike Clendenen, A.A.S., IC3
 Director of Distance Learning and Instructional Technology
Pitt Community College-VEW 20
P O Drawer 7007
Greenville, NC 27835-7007
Phone: 252-493-7608
 Email: mwclendenen091@my.pittcc.edu

**Comment by gjb2048** [comment]
Depending on fully understanding the problem with a screen shot, then probably around 30 mins with testing = 20 GBP.

**Comment by pccde** [comment]
That's reasonable.  I'll check with my management and see if we can "hire"
you for several enhancements if you are interested.

G'Day


Mike Clendenen, A.A.S., IC3
 Director of Distance Learning and Instructional Technology
Pitt Community College-VEW 20
P O Drawer 7007
Greenville, NC 27835-7007
Phone: 252-493-7608
 Email: mwclendenen091@my.pittcc.edu

**Comment by gjb2048** [comment]
Thanks Mike and I am interested.

**Comment by pccde** [comment]
I have asked the Vice President of IT how he wants to set this up.  I think
it would just require receiving an invoice from you.  Is that what you have
done in the past?


Mike Clendenen, A.A.S., IC3
 Director of Distance Learning and Instructional Technology
Pitt Community College-VEW 20
P O Drawer 7007
Greenville, NC 27835-7007
Phone: 252-493-7608
 Email: mwclendenen091@my.pittcc.edu

**Comment by gjb2048** [comment]
Hi Mike,

I can issue invoices from PayPal.

Kind regards,

Gareth

**Comment by pccde** [comment]
Please send a quote with a statement of work and we can go from there.  I
guess it might be more efficient to have you list several things at one
time to reduce paperwork?


Mike Clendenen, A.A.S., IC3
 Director of Distance Learning and Instructional Technology
Pitt Community College-VEW 20
P O Drawer 7007
Greenville, NC 27835-7007
Phone: 252-493-7608
 Email: mwclendenen091@my.pittcc.edu

**Comment by gjb2048** [comment]
Hi Mike,

Umm, I've already stated the quote for the work as above, being for this issue '"Assign Roles" icon in the Essential theme looks like a "lock"' to change to a user with a plus icon.  As the job is so small, I'm willing to complete it and then invoice once the work is done.  As the work is expected to take between 30 mins and 1 hour, I'll let you know what the actual time was once done with the 1 hour maximum.  If its less then I'll charge less.

What other things do you want done such that there would be more than one item on the quote?

If you need a full blown formal quote beyond what is stated already here then I may have to factor that in to the price and increase it.

Kind regards,

Gareth

**Comment by pccde** [comment]
I can understand that.  I'll see if I can get the forms you will need.
Since we are a state agency, a lot of paperwork is required.


Mike Clendenen, A.A.S., IC3
 Director of Distance Learning and Instructional Technology
Pitt Community College-VEW 20
P O Drawer 7007
Greenville, NC 27835-7007
Phone: 252-493-7608
 Email: mwclendenen091@my.pittcc.edu

**Comment by gjb2048** [comment]
Forms!!! It's a half hour job!  The invoices I can issue have full details with rate and addresses etc.  I have had no problems in the past with other businesses.  If there needs to be something specific then I do have an invoice template I can adapt.

I'm sorry but if there is a 'lot' of paperwork beyond what I normally do then I will have to factor that in and increase the price.

**Comment by pccde** [comment]
I am with you on that.  I have done contract work myself and the paperwork
is ridiculous!  I have no problem you charging a "processing fee" or
something similar.  I am going to talk with the purchasing department and
see if we cannot make this a simpler process.

I'll let you know.


Mike Clendenen, A.A.S., IC3
 Director of Distance Learning and Instructional Technology
Pitt Community College-VEW 20
P O Drawer 7007
Greenville, NC 27835-7007
Phone: 252-493-7608
 Email: mwclendenen091@my.pittcc.edu

**Comment by pccde** [comment]
Noticed that the Show/Hide icon for course content does not change when used.  However, the same icon for blocks works fine:

![show-hide](https://cloud.githubusercontent.com/assets/12518238/8115818/09d4da72-104c-11e5-9ee0-71482cba9d6d.png)

**Comment by gjb2048** [comment]
Hi Mike,

For clarity, please raise that as a separate issue otherwise things will get very VERY confusing.  Plus, what happens in Clean?

Gareth

**Comment by gjb2048** [comment]
No information in many months, closing.
